### PR TITLE
[WIP] Move second call omniture data inline

### PR DESCRIPTION
**Pull request by dominickendrick**
This is WIP @sndrs

### CODE REVIEW

File: common/app/templates/inlineJS/blocking/analytics.scala.js [hunk 1]
```diff
@@ -9,8 +9,22 @@ try {
...
             tpA     = s.getTimeParting('n', '+0'),
-            now      = new Date(),
-            webPublicationDate = config.page.webPublicationDate;
+            now     = new Date(),
+            webPublicationDate = config.page.webPublicationDate,
+            w       = window,
+            Storage = function (type) {
+                this.type = type;
+            },
+            storage.local = new Storage('localStorage'),
```
Comment (desbo) [inline common/app/templates/inlineJS/blocking/analytics.scala.js]: is `storage` defined as a global somewhere else?

File: common/app/templates/inlineJS/blocking/analytics.scala.js [hunk 2]
```diff
@@ -21,6 +35,46 @@ try {
...
+            var data = [];
+            for (var i = 0; i < array.length; i++) {
+                if(data.indexOf(array[i]) == -1) {
+                    data.push(array[i]);
+                }
+            }
+            return data;
+        }
+
+        function getUserFromCookie() {
```
Comment (sndrs) [inline common/app/templates/inlineJS/blocking/analytics.scala.js]: this functionality is duplicated in `common/app/templates/inlineJS/nonBlocking/getUsername.scala.js` - should extract one out and share

**Comment by sndrs** [comment]
`blocking/analytics.scala.js` should probably be moved to `nonBlocking/analytics.scala.js`?

**Comment by sndrs** [comment]
>blocking/analytics.scala.js should probably be moved to nonBlocking/analytics.scala.js? 

done in bca9642

**Comment by johnduffell** [comment]
5 weeks must be a record! :stuck_out_tongue: is this PR real or can it be closed until it's ready?

**Pull request closed without merging**